operator/metrics.go

```diff
@@ -38,6 +73,60 @@ func (l *SelectiveRpcClientListener) OnMessagesReceived() {
 	}
 }
 
+func (l *SelectiveRpcClientListener) ObserveResendQueueSize(size int) {
+	if l.ObserveResendQueueSizeCb != nil {
+		l.ObserveResendQueueSizeCb(size)
+	}
+}
+
+func (ol *SelectiveRpcClientListener) ObserveLastCheckpointIdResponded(checkpointId uint32) {
```

Comment:
```suggestion
func (l *SelectiveRpcClientListener) ObserveLastCheckpointIdResponded(checkpointId uint32) {
```

Receiver names a different on `SelectiveRpcClientListener`. All of the below

Reply:
Done in [f8cf481](https://github.com/NethermindEth/near-sffl/pull/159/commits/f8cf4813c8b8932e47c02af0d1848362ca262130)